xc/device.ml

```diff
@@ -1493,7 +1493,15 @@ module Dm = struct
 
     let qmp_event_handle domid qmp_event =
       (* This function will be extended to handle qmp events *)
-      debug "Got QMP event, domain-%d: %s" domid qmp_event.event
+      debug "Got QMP event, domain-%d: %s" domid qmp_event.event;
+      qmp_event.data >>= fun d ->
+      match d with
+      | RTC_CHANGE timeoffset ->
+        with_xs (fun xs ->
+          let timeoffset_key = "/vm/" ^ (Uuidm.to_string (Xenops_helpers.uuid_of_domid ~xs domid)) ^ "/rtc/timeoffset" in
```

Comment:
It think it would be clearer to use `sprintf "/vm/%s/rtc/timeoffset" (...)`

Reply:
Done, thanks.